src/test/java/uk/gov/ons/census/casesvc/messaging/ReceiptReceiverIT.java

```diff
@@ -134,6 +135,11 @@ public void testReceiptEmitsMessageAndEventIsLoggedForCase() throws Exception {
           responseManagementEvent.getPayload().getCollectionCase();
       assertThat(actualCollectionCase.getId()).isEqualTo(TEST_CASE_ID.toString());
       assertThat(actualCollectionCase.getReceiptReceived()).isTrue();
+      assertThat(actualCollectionCase.getCreatedDateTime()).isEqualTo(caze.getCreatedDateTime());
+
+      Optional<Case> updatedCase = caseRepository.findById(caze.getCaseId());
+      assertThat(actualCollectionCase.getLastUpdated())
+          .isEqualTo(updatedCase.get().getLastUpdated());
```

Comment:
This seems like a weird test to add this too?

Reply:
Tbh I could have added this to any integration test which should save and emit the date/time but just randomly chose this one. The integration tests check for fields that are expected to be there rather than checking for new ones meaning none of the tests fail if you create new fields so I just wanted to assert that the createdDateTime and lastUpdated fields were being checked for. Do you think there is a better place to put this?

Reply:
I recommended putting the check here, because we need an integration test to check that the last updated value has been stamped by Postgres. It didn't seem to warrant an integration test all of its own.

Reply:
I think it should either have its own integration test if it's worth it (don't think it is) or be kept to unit tests.

Reply:
Perhaps there should be an IT for emitting a case updated event as a whole that test all the fields then? I'm not keen on tacking it on in one random IT.